.github/workflows/test-microk8s-aws.yaml

```diff
@@ -0,0 +1,95 @@
+# Runs tests on AWS that are too big for github actions
+# Tests that are small enough are in test-microk8s.yaml
+
+name: MicroK8s Test
```

Comment:
Might want to make this different from the other workflow so that we can tell the results apart.

Reply:
I just realized that I should just put them as multiple jobs in the same file, will update that